azure_functions/local.settings.json

```diff
@@ -6,7 +6,7 @@
     "AzureWebJobs.HttpExample.Disabled": "true",
     "AZURE_STORAGE_CONNECTION_STRING": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:12000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:12001/devstoreaccount1;",
     "POLLING_TRIGGER_QUEUE_NAME": "polling-trigger-queue",
-    "CA_DPH_POLLING_CRON": "0 30 9 * Jan Mon"
+    "CA_DPH_POLLING_CRON": "0 */1 * * * *"
```

Comment:
Should we add a comment explaining why this is different from the other environments or is it too rudimentary?

Reply:
json isn't allowed to have comments, but this CRON will run every minute, on the minute, and only in the `local` environment